src/components/Vault/BasicVault/CardHeader.tsx

```diff
@@ -50,7 +48,8 @@ const CardHeader = ({
         <Col className="col-item" lg={{ span: 4 }} md={{ span: 6 }} xs={{ span: 12 }}>
           <div className="title">Reward Pool</div>
           <div className="content">
-            {commaFormatter(totalPendingRewards.dp(4, 1).toFixed())} {rewardToken}
+            {formatCommaThousandsPeriodDecimal(totalPendingRewards.dp(4, 1).toFixed())}{' '}
```

Comment:
shouldn't we have a separate util for this case

Reply:
for wrapping like `toSomeFormat(totalPendingRewards)`

Reply:
We could. The requirements for formatting are a varied once you get outside of setting decimals places and casting to a string. We'd probably need several functions to handle those cases. For this PR I focused on wrapping our use of `comma-number`